plugins/fireeye_hx/icon_fireeye_hx/util/api.py

```diff
@@ -56,6 +57,20 @@ def get_host_id_from_hostname(self, params: dict) -> requests.Response:
     def get_version(self) -> requests.Response:
         return self.call_api(endpoint.version())
 
+    def check_host_quarantine_status(self, agent_id: str) -> requests.Response:
+        return self.call_api(endpoint.host_containment(agent_id)).json()
+
+    def quarantine_host(self, agent_id: str) -> bool:
+        action_endpoint = endpoint.host_containment(agent_id)
+        self.call_api(action_endpoint, "POST")
+        time.sleep(5)
+        self.call_api(action_endpoint, "PATCH", json_data={"state": "contain"})
```

Comment:
I'm assuming the sleep is here because the API needs some time to process the initial POST call, and then to PATCH it later. Is 5 seconds a documented interval to sleep? If not, is there a chance that 5 seconds is not enough? Should this include a retry with backoff to ensure quarantining works properly all the time?